UnityMcpBridge/UnityMcpServer~/src/telemetry.py

```
@@ -0,0 +1,373 @@
+                self._milestones = json.loads(content) or {}
+                if not isinstance(self._milestones, dict):
+                    self._milestones = {}
+        except (OSError, json.JSONDecodeError, ValueError) as e:
+            logger.debug(f"Failed to load milestones: {e}", exc_info=True)
+            self._milestones = {}
+    
+    def _save_milestones(self):
+        """Save milestones to disk"""
+        try:
+            self.config.milestones_file.write_text(json.dumps(self._milestones, indent=2))
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)